src/connector/ui.jsx

```diff
@@ -836,4 +862,110 @@ Zotero.GoogleDocs.UI.PleaseWait = class extends React.Component {
 	}
 }
 
+
+Zotero.GoogleDocs.UI.OrphanedCitations = React.forwardRef(function(props, ref) {
+	let [open, setOpen] = React.useState(false);
+	let [citations, setCitations] = React.useState([]);
+	
+	React.useImperativeHandle(ref, () => ({
+		setCitations: function(newCitations) {
+			// Open upon first detecting orphaned citations
+			if (!citations.length && newCitations.length) setOpen(true);
+			setCitations(newCitations);
+		}
+	}));
+
+	// A bit dirty here. Injecting a hover CSS rule. The other option is to
+	// do it via background scripts, which would be a million lines of code
+	// especially since we eval-load gdocs code on browserExt.
+	const buttonID = "zotero-docs-orphaned-citations-button";
+	const hoverRule = `#${buttonID}:hover {background: rgba(0, 0, 0, .06);}`;
+	let styleElem;
+	React.useEffect(() => {
+		styleElem = document.createElementNS("http://www.w3.org/1999/xhtml", 'style');
+		document.head.appendChild(styleElem);
+		styleElem.sheet.insertRule(hoverRule);
+		return () => {
+			document.head.removeChild(styleElem);
+		}
+	}, []);
+
+	React.useEffect(() => {
+		let listener = () => open && setOpen(false);
+		document.addEventListener('click', listener, {capture: true});
+		return () => {
+			document.removeEventListener('click', listener);
+		}
+	}, [open]);
+
+	return (
+		<div
+			role={"button"}
+			id={buttonID}
+			dataTooltip="Open unlinked citation list"
+			ariaLabel="Open unlinked citation list"
```

Comment:
1) Can we localize these?

2) These aren't actually working properly for me — they're coming through in the DOM as `datatooltip` and `ariatooltip`.

Reply:
These actually don't seem to work, at least as google docs tooltips. I think I'd have to inject most of this UI before page load and react being available, and idk what will happen if we start hiding that button manually, if we allow Google Docs to attach its JS to it. I don't think not having a tooltip for that button is a huge deal for the UI element that most people won't see, and when they do, will immediately get alerted to what's it about. The aria label, if fixed, might work for screenreaders, so I'll look into that.

Reply:
I got a tooltip when I changed it in the devtools.